### ISSUE TITLE
Улучшение дизайна

### ISSUE THREAD

**Issue by egorpugachevskyWork** [Katletos/GhostHost#3]
Для максимизации удовлетворения от использования данного сайта следует сделать следующее: при наведении курсора на кнопки "Login" & "Register" нужно подсвечивать данные кнопки!!!
[Click](https://codepen.io/dmensinger/pen/PPRzpK/)

**Comment by damir-abdulin** [comment]
Согласен, но мне кажется, что ваш пример не сильно улучшит внешний вид сайта. Нужно искать что-то другое
